packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -338,6 +374,12 @@ export class OdspDelayLoadedDeltaStream {
 				isRefreshingJoinSession,
 				this.hostPolicy.sessionOptions?.unauthenticatedUserDisplayName,
 			);
+			// Emit event only in case it is fetched from the network.
+			if (joinSessionResponse.sensitivityLabelsInfo !== undefined) {
+				this.emitMetaDataUpdateEvent({
```

Comment:
Similar comment here - let's make sure we correctly handle undefined

Reply:
Refer to this: https://github.com/microsoft/FluidFramework/pull/19142#discussion_r1446619872